### PR TITLE
Refactor porter bundles to simplify dependencies.

### PR DESCRIPTION
**Pull request by marrobi**
# PR for issue #350

## What is being addressed

Refactor porter bundles to simplify dependencies.

## How is this addressed

- Move to using scripts as proved out with the innereye inference bundle
- Remove workspace outputs no longer needed.

### CODE REVIEW

File: workspaces/azureml_devtestlabs/install_workspace_vanilla.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -e
+
+porter install tre-workspace-vanilla --reference "${ACR_NAME}.azurecr.io/tre-workspace-vanilla:v0.1.0" \
```
Comment (sachinkundu) [inline workspaces/azureml_devtestlabs/install_workspace_vanilla.sh:4]: consider making the code DRY as the file above is almost the same and exactly the same if the parameters are abstracted as inputs

**Review by sachinkundu — COMMENTED** [review]
I think you should write one bash script which takes args and does what you need done instead of replicating the same commands in many files. You can make a dict of images and parameters etc and then only one place needs to be changed when eventually changes do happen.

**Comment by marrobi** [comment]
@sachinkundu agree, although would still need to be one in each bundle as are separate docker contexts.

The challenge comes with when need to retrieve different outputs for different bundles. Any suggestions?

I'm hoping these scripts becomes redundant once we have the API configured to deploy services.

**Comment by sachinkundu** [comment]
> @sachinkundu agree, although would still need to be one in each bundle as are separate docker contexts.
> 
> The challenge comes with when need to retrieve different outputs for different bundles. Any suggestions?
> 
> **I'm hoping these scripts becomes redundant once we have the API configured to deploy services.**
Hence approved.

**Review by sachinkundu — APPROVED** [review]

**Pull request merged**